torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1979,7 +1964,24 @@ void ONNXShapeTypeInference(
     for (auto output : clone_node->outputs()) {
       n_graph->registerOutput(output);
     }
-
+    // Use outputs_map for original PyTorch output name to corresponding ONNX output name
+    for (size_t i = 0; i < clone_node->outputs().size(); ++i) {
+      outputs_map[n->output(i)->debugName()] = clone_node->output(i)->debugName();
+    }
+    // Make generated_shape_data use name from ONNX graph instead of original PyTorch graph
+    // Use original_keys for removing original data which is duplicate
+    std::vector<string> original_keys;
+    for (auto gs_data: generated_shape_data) {
+      if (outputs_map.count(gs_data.first) > 0) {
```

Comment:
Given that you refer to `outputs_map[gs_data.first]` 3 times in a short span, I think it may be worth while to assign it to a local variable for readability and avoiding lots of look-ups. Something like:
```suggestion
      const auto onnx_output_name = outputs_map.find(gs_data.first);
      if (onnx_output_name != outputs_map.end()) {
```

And then you can use `onnx_output_name` below too.

Reply:
I have used onnx_output_name to make it clear. Thanks!